src/Microsoft.Health.Fhir.Core/Features/Operations/Export/Models/ExportJobRecord.cs

```diff
@@ -25,6 +25,7 @@ public ExportJobRecord(Uri exportRequestUri)
             SchemaVersion = 1;
             Status = OperationStatus.Queued;
             Id = Guid.NewGuid().ToString();
+            SecretName = Id;
```

Comment:
Should give this a bit more meaningful name. A bunch of GUID in the KeyVault might be hard to tell what is what.

Reply:
Added a prefix to id - "Export_Destination_"